src/run.php

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env php
+<?php
+// src/run.php
+
+require __DIR__ . '/../vendor/autoload.php';
+
+use Consolidation\AnnotatedCommand\AnnotatedCommandFactory;
+use Consolidation\AnnotatedCommand\CommandFileDiscovery;
+
+$setAutoloader();
+$terminus = new Terminus();
+$command_factory = new AnnotatedCommandFactory();
+//$formatter_manager = new FormatterManager();
+
+$discovery = new CommandFileDiscovery();
+$command_files = $discovery->discover(__DIR__, '\Commands');
+foreach ($command_files as $command_class) {
+  $command_instance = new $command_class();
+  //$command_factory->commandProcessor()->setFormatterManager($formatter_manager);
+  $command_list = $command_factory->createCommandsFromClass($command_instance);
+  foreach ($command_list as $command) {
+    $terminus->add($command);
+  }
+}
+
+/**
+  * Sets the autoloader up to retrieve files as necessary
+  *
+  * @return void
+  */
+function setAutoloader() {
```

Comment:
@greg-1-anderson Although I considered doing that, I chose against it. The reason being is that many Terminus users install via methods which will automatically add the 1.x work to users' files when updating meaning we don't have to make the users install a second application in order to rope in alpha testers.